### ISSUE TITLE
[Question] How can I get ASR timestamps?

### ISSUE THREAD

**Issue by ValeryNikiforov** [NVIDIA/NeMo#2463]
Is there any way to get timestamps for Citrinet output logits?
It's important for my experiments, thanks in advance.

**Comment by titu1994** [comment]
Currently we do not support it, but the offline asr tutorial for ASR does have some discussion of getting audio timestamps which can be ported to Citrinet. Just note that Citrinet had 8x stride so many parameters in that notebook will need to he changed to support such inference c
